lib/db.js

```diff
@@ -594,6 +594,30 @@ module.exports = (
     })
   }
 
+  DB.prototype.updateSessionToken = function (sessionToken, geo) {
```

Comment:
Since we now need to call the `updateSessionToken` method over in auth-db-mysql, I opted to rename the existing redis-only version of `updateSessionToken` in order to make room for this one.  It would be neat to have a single `updateSessionToken` method that can choose whether to write back to the db depending on what is being updated, but I couldn't see a nice way for it to determine which fields where updates and which still had their old values.